### PR TITLE
updated airmail-beta (3.0,364)

### PR DESCRIPTION
**Pull request by WardLootens**
#### Editing an existing cask

- [x] Commit message includes cask’s name (and new version, if applicable).
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.

### CODE REVIEW

File: Casks/airmail-beta.rb [hunk 1]
```diff
@@ -1,11 +1,11 @@
 cask 'airmail-beta' do
-  version '2.6.1,363'
-  sha256 '768f3ed8fedc93f0a8438bab7d5080607fb73c5fbad5ef487b119602606c2487'
+  version '3.0,364'
```
Comment (vitorgalvao) [inline Casks/airmail-beta.rb]: `3.0,245`

File: Casks/airmail-beta.rb [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+  sha256 '447ce9fe31cdeab0c4e9ce7fb9d7ee6b44b47e3bdaf12b8abf541c4cf21472dd'
 
   # hockeyapp.net/api/2/apps/84be85c3331ee1d222fd7f0b59e41b04 was verified as official when first introduced to the cask
-  url 'https://rink.hockeyapp.net/api/2/apps/84be85c3331ee1d222fd7f0b59e41b04/app_versions/244?format=zip&'
+  url 'https://rink.hockeyapp.net/api/2/apps/84be85c3331ee1d222fd7f0b59e41b04/app_versions/245?format=zip&'
```
Comment (vitorgalvao) [inline Casks/airmail-beta.rb]: `url 'https://rink.hockeyapp.net/api/2/apps/84be85c3331ee1d222fd7f0b59e41b04/app_versions/#{version.after_comma}?format=zip&'`

**Comment by WardLootens** [comment]
Good idea to make download link dependent on version number. I added an extra commit.

**Comment by vitorgalvao** [comment]
Looks like there are conflicts, now.

**Comment by fanquake** [comment]
@WardLootens Would you like to rebase this? Also, is the new version 3 different from version 2 somehow? Should it be a different cask?

**Comment by WardLootens** [comment]
Unfortunately not that much time these days to review this change. I will close this pull request and open a new one later on. I don't really think this should become a new cask. Unclear to me why major version number increased with this (at first sight) common update...

**Pull request closed without merging**